src/bluetooth/bluez/obextransfer.cpp

```diff
@@ -0,0 +1,116 @@
+/****************************************************************************
+**
+** Copyright (C) 2015 Jolla Ltd.
+** Contact: Hannu Mallat <hannu.mallat@jollamobile.com>
```

Comment:
Copy the Contact: lines from the other files, they all need to point to the web URL to pass license checks.